documentation/custom-image-support/ade-cli-docs/outputs.md

```diff
@@ -0,0 +1,54 @@
+# 'ade outputs' Command Set
+The 'ade outputs' command allows a customer to upload outputs from the deployment of an Infrastructure-as-Code (IaC) template to be accessed from the Outputs API for ADE. 
+
+## 'ade outputs upload' 
+This command uploads the contents of a JSON file specified in the ADE EnvironmentOutput format to the environment, to be accessed later using the Outputs API for ADE.
+
+### Options
+**--file**: A file location containing a JSON object to upload.
+
+### Examples
+
+This command uploads a .json file named 'outputs.json' to the environment to serve as the outputs for the successful deployment:
+```
+ade outputs upload --file outputs.json
+```
+
+### EnvironmentOutputs Format
+In order for the incoming JSON file to be serialized properly and accepted as the environment's deployment outputs, the object submitted must follow the below structure:
+```
+{
+    "outputs": {
+        "output1": {
+            "type": "string",
+            "value": "This is output 1!",
+            "sensitive": false
+        },
+        "output2": {
+            "type": "int",
+            "value": 22,
+            "sensitive": false
+        },
+        "output3": {
+            "type": "string",
+            "value": "This is a sensitive output",
+            "sensitive" true
+        }
+    }
+}
+```
+
+This format is adapated from how ARM deployments report outputs of a deployment, along with an additional property of "sensitive". The "sensitive" property is optional to provide, but will restrict the output to only being able to be viewed by those with privileged access, such as the creator of the environment.
```

Comment:
Is this true? We do have an RBAC action for reading outputs, but nothing for viewing sensitive outputs as far as I know.

Reply:
I'm not sure, I assumed that was the idea behind this feature, but I'd need to follow up with Aaron about it's usage and how we should document it